shared/common-adapters/plain-input.desktop.tsx

```diff
@@ -175,8 +175,8 @@ class PlainInput extends React.PureComponent<InternalProps> {
 
   _getMultilineProps = () => {
     const rows = this.props.rowsMin || Math.min(2, this.props.rowsMax || 2)
-    const textStyle = getTextStyle(this.props.textType)
-    const heightStyles: _StylesDesktop = {
+    const textStyle: any = getTextStyle(this.props.textType)
+    const heightStyles: any = {
```

Comment:
what happened here?

Reply:
I didn't want to correctly type getTextStyle right now